Content.Client/UserInterface/Screens/SeparatedChatGameScreen.xaml.cs

```diff
@@ -36,10 +60,53 @@ private void ResizeActionContainer()
         Actions.ActionsContainer.MaxGridWidth = ViewportContainer.Size.X - indent;
     }
 
+    private void RecalculateViewportDesiredSize()
+    {
+        // When the main window is reszied, calculate a new min/max for the
+        // ViewportContainer so that the chat will prefer to resize instead
+        // of letterboxing the viewport.
+        var uiScale = _cfg.GetCVar(CVars.DisplayUIScale);
+        if (uiScale == 0f)
+            uiScale = _uiManager.DefaultUIScale;
+
+        // CurrentRenderScale would be more correct, but it gets updated after us.
+        var gameScale = MainViewport.Viewport.FixedRenderScale;
+        var minFromVp = EyeManager.PixelsPerMeter * gameScale * _cfg.GetCVar(CCVars.ViewportMinimumWidth) / uiScale;
+        var maxFromVp = EyeManager.PixelsPerMeter * gameScale * _cfg.GetCVar(CCVars.ViewportMaximumWidth) / uiScale;
+        // Inventory bar also supplies a minWidth such that body+shoes don't overlap PDA.
+        var minFromUi = 1060;
```

Comment:
Can you tell us exactly this comes from? 

If you're gonna define constants, it would probably be better to actually make them such and document exactly where they come from (e.g. manually collected).